common/authen.js

```diff
@@ -20,6 +20,148 @@
                 _changeCbs[k]();
             }
         };
+        
+        var popUpLoginCb = function(params, referrerId, cb){
```

Comment:
This function and function below it share a lot of code. Can we merge them together? Instead of having two different function, we can have a function that as a parameter will determine to open a modal or not.